gateway/app.py

```diff
@@ -28,7 +28,10 @@
 from models import EndpointApp, OnboardingAppKey
 from util import make_hash
 from app_factory import create_app
-
+# pylint: disable-next=unused-import
+from scim_ethermab import EtherMABExtension
+# pylint: disable-next=unused-import
+from scim_fdo import FDOExtension
```

Comment:
Why is this required?

Reply:
Because all database objects have to be declared so they can be instantiated in db.create_all.
This was driving me BONKERS until I figured it out.  And pytest won't spot it either.